cmd/sidecarinjector/main.go

```diff
@@ -33,13 +33,9 @@ const (
 )
 
 func main() {
-	// This line is necessary to make glog happy
-	// TODO: move away from glog
-	flag.CommandLine.Parse([]string{})
-
 	webhookConfig, err := config.NewWebhookConfig()
 	if err != nil {
-		glog.Errorf("api=main, reason=config.NewWebhookConfig, err=%v", err)
+		fmt.Fprintf(os.Stderr, "api=main, reason=config.NewWebhookConfig, err=%v\n", err)
```

Comment:
Note: changed to use `fmt.Fprintf` here since we are not guaranteed that `glog` has been initialized in this path.